doc/plugins/org.polarsys.capella.doc/html/Release note/API_5.x_to_6.x.html

```diff
@@ -15,10 +15,8 @@ <h3 id="Migration_to_Eclipse_2021-06">Migration to Eclipse 2021-06</h3>
 		<p>Capella 6.x is built with Eclipse 2021-06.</p>
 		<h3 id="Version_major_increased">Version major increased</h3>
 		<p>Metamodels have been updated to 6.0.0</p>
-		<h3 id="A_JRE_is_now_embedded">A JRE is now embedded</h3>
-		<p>A 
-			<b>JRE</b> Temurin Java 17 is now installed instead of a JDK under the jre folder in the capella folder.
-		</p>
+		<h3 id="JDK_upgraded">JDK upgraded</h3>
```

Comment:
can you also update the table of content (https://github.com/eclipse/capella/blob/5b055350da7456d89fb4de78908bb48f7fb1001a/doc/plugins/org.polarsys.capella.doc/toc.xml#L24 )